interop/tapir/src/main/scala/caliban/interop/tapir/package.scala

```diff
@@ -125,7 +125,7 @@ package object tapir {
                 val replacedArgs = args.map { case (k, v) => reverseArgNames.getOrElse(k, k) -> v }
                 QueryStep(
                   ZQuery
-                    .fromZIO(ZIO.fromEither(argBuilder.build(InputValue.ObjectValue(replacedArgs))))
+                    .fromZIO(Exit.fromEither(argBuilder.build(InputValue.ObjectValue(replacedArgs))))
```

Comment:
There's no `ZQuery.fromEither` btw?

Reply:
Ah yeah I added that recently, I'll change it